boards/shields/reyax_lora/reyax_lora.overlay

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2024 David Ullmann
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/
```

Comment:
extra `/`?

Reply:
ahh it just belongs after the #include (but must go before the root node)